exp/runtime/api/v1beta1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1beta1 contains API Schema definitions for the exp runtime v1beta1 API group
+// +kubebuilder:object:generate=true
+// +groupName=runtime.cluster.x-k8s.io
+package v1beta1
```

Comment:
Should we add this as v1beta1 (the same API version as the core group) even though it's a completely new API group (and behind an alpha feature flag) ?

Reply:
I think I'm definitely leaning toward having this at v1alpha1 initially, but it would be good to source more opinions on this